src/main/webapp/dev/js/common/updateCharCount.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Updates the number of characters left in the text area
+ * @param textArea - Text area for which char are to be counted
+ * @param letterCountId - Id of Label to display length of text area
+ */
+function updateCharLeftCount(textArea, letterCountId) {
+    const letterCountArea = $(`#${letterCountId}`);
+    const maxLength = textArea.attr('maxLength');
+    const charLength = textArea.val().length;
+    letterCountArea.text(maxLength - charLength);
+}
+/**
+ * Create the div to display the number of characters left in textArea
+ * @param textArea - Text area for which char are to be counted
+ * @param letterCountId - Id of area for letter count display
+ */
+function insertLetterCountArea(textArea, letterCountId) {
+    const maxLength = textArea.attr('maxlength');
+    if ($(`#${letterCountId}`).length === 0) {
+        const letterCountAreaTemplate = `<div class="col-md-6 padding-0">
+            <span id="${letterCountId}">${maxLength}</span> characters left </div>`;
```

Comment:
Although I do like that the code snippet has been extracted into a template, I'm strangely uncomfortable with the fact that the template is 'hidden' here. I wonder if there is a more appropriate location to place this...?

Reply:
@LiHaoTan  what is ur opinion

Reply:
@whipermr5 can you help us take a look at this? some disagreement over whether we shld append html using js function here. 
What I want to do is to couple the maxLength attribute with the input, so that whenever the function is called over some input there will be a maxLength display. 
What @joanneong thinks is that we should not put html in js functions.

Reply:
@whipermr5  pls help us take a look at this thx!

Reply:
While it may not be ideal, I'm pretty sure this is done elsewhere. I don't have strong objections to it. @LiHaoTan do you have strong objections?

Reply:
I don't have any objections actually if this is thought of as a single component. The module name `updateCharCount` is a little strange though.